LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs

```diff
@@ -42,6 +43,7 @@ internal sealed class AmdGpu : Hardware
         private readonly Sensor _temperaturePlx;
         private readonly Sensor _temperatureSoC;
         private readonly Sensor _temperatureVddc;
+        private readonly Sensor _fullscreen_fps;
```

Comment:
Can you name this _fullscreenFPS to follow the rest?

Reply:
Sure, fixed